elements.html

```diff
@@ -87,23 +98,19 @@ <h3 class="s-h3">Savile Links</h3>
   <tbody>
     {% assign links = site.elements | where: "element_type", "link" %}
     {% for link in links %}
-    <tr class="token-table-row">
-      <td class="element-code-cell">
-        <code>{{ link.value }}</code>
-      </td>
-      <td>
-        <code class="language-html" data-lang="html">
-            {{link | remove: '<p>' | remove: '</p>' | escape}}
-          </code>
-      </td>
+    <tr class="table-row">
+      <td class="element-class-cell">{% highlight html %}{{ link.value }}{% endhighlight %}</td>
+      <td class="element-code-cell">{% highlight html %}{{ link | remove: '<p>' | remove: '</p>' }}{% endhighlight %}
       </td>
       <td class="element-sample-cell">{{ link }}</td>
     </tr>
     {% endfor %}
   </tbody>
 </table>
 
-<h2 class="s-h2">Paragraph Text</h2>
+<a id="paragraphs">
+  <h2 class="s-h2">Paragraphs</h2>
+</a>
```

Comment:
Same as above - can we just use `id` on the `h2` element and avoid wrapping with `<a>`?